static/src/stylesheets/module/journalism/_campaigns.scss

```diff
@@ -285,24 +304,12 @@
     background-color: $news-main;
 }
 
-.speech-bubble {
-    position: relative;
 
-    &::after {
-        content: '';
-        width: 20px;
-        height: 22px;
-        border-radius: 0 0 18px;
-        position: absolute;
-        bottom: -12px;
-        left: 10px;
-    }
-}
 
 /*---------- SUCCESS STATE TOGGLE -----------------*/
 
 .success.element-campaign .campaign--snippet {
-    .heading {
```

Comment:
I guess this selector was unique to the callout element?

Reply:
Well, because it's wrapped in the `.campaign--snippet` element I felt safe to leave it like that. But happy to make it `campaign--snippet_heading` if you prefer?

Reply:
will change